src/__experimental__/components/decimal/decimal.stories.mdx

```diff
@@ -0,0 +1,270 @@
+
+## Quick Start
+
+```jsx
+import Decimal from "carbon-react/lib/__experimental__/components/decimal";
+```
+
+### Default
+
+<Preview>
+  <Story name="default">
+    {() => {
+      const [state, setState] = useState("0.01");
+      const setValue = ({ target }) => {
+        setState(target.value.rawValue);
+      };
+      return <Decimal label="Decimal" value={state} onChange={setValue} />;
+    }}
+  </Story>
+</Preview>
+
+### Sizes
+
+<Preview>
+  <Story name="sizes">
+    {() =>
+      ["small", "medium", "large"].map((size) => (
+        <Decimal
+          key={`Decimal - ${size}`}
+          label={`Decimal - ${size}`}
+          value="0.01"
+          size={size}
+          mb={2}
+        />
+      ))
+    }
+  </Story>
+</Preview>
+
+### Disabled
+
+<Preview>
+  <Story name="disabled">
+    <Decimal label="Decimal" value="0.01" disabled />
+  </Story>
+</Preview>
+
+### LabelAlign
+
+<Preview>
+  <Story name="labelAlign">
+     {() => 
+      ['right', 'left'].map((alignment) => (
+        <Decimal
+          label="Decimal"
+          labelInline
+          inputWidth={50}
+          key={alignment} 
+          labelAlign={alignment}
+          value="0.01"
+        />
+      ))
+    }
+  </Story>
+</Preview>
+
+### ReadOnly
+
+<Preview>
+  <Story name="readOnly">
+    <Decimal label="Decimal" value="0.01" readOnly />
+  </Story>
+</Preview>
+
+### Empty
+
+<Preview>
+  <Story name="empty">
+    <Decimal label="Decimal" allowEmptyValue />
+  </Story>
+</Preview>
+
+### With custom precision
+
+<Preview>
+  <Story name="with custom precision">
+    <Decimal label="Decimal" value="0.0001" precision={4} />
+  </Story>
+</Preview>
+
+### With labelInline
+
+<Preview>
+  <Story name="with labelInline" parameters={{ chromatic: { disable: true }}}>
+    <Decimal label="Decimal" value="0.01" labelInline />
+  </Story>
+</Preview>
+
+### With custom labelWidth and inputWidth
+
+<Preview>
+  <Story name="with custom labelWidth and inputWidth">
+    <Decimal
+      label="Decimal"
+      value="0.01"
+      labelInline
+      labelWidth={10}
+      inputWidth={90}
+    />
+  </Story>
+</Preview>
+
+### With fieldHelp
+
+<Preview>
+  <Story name="with fieldHelp">
+    <Decimal label="Decimal" value="0.01" fieldHelp="Help" />
+  </Story>
+</Preview>
+
+### With labelHelp
+
+<Preview>
+  <Story name="with labelHelp">
+    <Decimal label="Decimal" value="0.01" labelHelp="Help" />
+  </Story>
+</Preview>
+
+### Required
+
+You can use the `required` prop to indicate if the field is mandatory.
+
+<Preview>
+  <Story name="required">
+    <Decimal label="Decimal" value="0.01" required />
+  </Story>
+</Preview>
+
+### Left aligned
+
+You can use the `required` prop to indicate if the field is mandatory.
+
+<Preview>
+  <Story name="left aligned">
+    <Decimal label="Decimal" value="0.01" align="left" />
+  </Story>
+</Preview>
+
+## Validations
```

Comment:
The validation stories should be combined so that the cypress tests that check for accessibility can find the right page

Reply:
This has been fixed on cypress side